### PR TITLE
Capistrano production recipe

### PR DESCRIPTION
**Pull request by egorodessa**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -15,6 +15,10 @@ gem 'sass-rails', '~> 5.0'
 gem 'slim'
 gem 'uglifier', '>= 1.3.0'
 
+gem 'slim'
```
Comment (olehif) [inline Gemfile]: duplicate

File: Gemfile [hunk 2]
```diff
@@ -34,3 +38,12 @@ group :development do
 end
 
 gem 'tzinfo-data', '~> 1.2017', '>= 1.2017.3'
+
+group :production do
```
Comment (olehif) [inline Gemfile]: move to development group

File: config/database.yml.example [hunk 3]
```diff
@@ -1,87 +0,0 @@
-
```
Comment (olehif) [inline config/database.yml.example]: you need this file

File: config/secrets.yml.example [hunk 4]
```diff
@@ -1,32 +0,0 @@
-# Be sure to restart your server when you modify this file.
```
Comment (olehif) [inline config/secrets.yml.example]: you need this file

File: test/fixtures/tests.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (olehif) [inline test/fixtures/tests.yml]: factorybot

File: test/models/test_test.rb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (olehif) [inline test/models/test_test.rb]: It's empty + you have rspec in your gemfile, so use it

File: config/deploy.rb [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+set :application, "heroes_team_app"
+set :repo_url, "git@github.com:Ruby-IF081/heroes_team_app.git"
+
+# Default branch is :master
+ask :branch, :master # `git rev-parse --abbrev-ref HEAD`.chomp
+
+# Default deploy_to directory is /var/www/my_app_name
+# set :deploy_to, "/var/www/my_app_name"
+set :deploy_to, "/var/www/heroes_team_app/code"
```
Comment (f1nwe) [inline config/deploy.rb]: are you sure, that you want deploy there?

File: config/deploy.rb [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+lock "~> 3.10.1"
+
+# set :application, "my_app_name"
+# set :repo_url, "git@example.com:me/my_repo.git"
+
+set :application, "heroes_team_app"
+set :repo_url, "git@github.com:Ruby-IF081/heroes_team_app.git"
+
+# Default branch is :master
+ask :branch, :master # `git rev-parse --abbrev-ref HEAD`.chomp
```
Comment (f1nwe) [inline config/deploy.rb]: it is better to set branch in production.rb

File: config/deploy.rb (hunk 7 above)
Comment (egorodessa) [reply config/deploy.rb]: sure, already

File: config/database.yml.example (hunk 3 above)
Comment (egorodessa) [reply config/database.yml.example]: done

File: config/secrets.yml.example (hunk 4 above)
Comment (egorodessa) [reply config/secrets.yml.example]: done

File: test/models/test_test.rb (hunk 6 above)
Comment (egorodessa) [reply test/models/test_test.rb]: deleted

File: test/fixtures/tests.yml (hunk 5 above)
Comment (egorodessa) [reply test/fixtures/tests.yml]: deleted

File: Gemfile (hunk 1 above)
Comment (egorodessa) [reply Gemfile]: solved

File: Gemfile (hunk 2 above)
Comment (egorodessa) [reply Gemfile]: done

File: config/deploy.rb (hunk 8 above)
Comment (egorodessa) [reply config/deploy.rb]: done

File: config/deploy.rb (hunk 7 above)
Comment (egorodessa) [reply config/deploy.rb]: done

Comment (Kmalynss) [reply config/deploy.rb]: I believe this now was not about deploy.rb/production.rb but it was about deployment path.
Why did you add ```/code``` directory?

Comment (egorodessa) [reply config/deploy.rb]: just followed the one of tutorial partly, and they used such directory structure

Comment (Kmalynss) [reply config/deploy.rb]: @egorodessa please remove that ```/code```

Comment (egorodessa) [reply config/deploy.rb]: done

**Pull request merged**